### ISSUE TITLE
Tolino Shine 5 (Kobo Clara BW equivalent), Firmware 5.x

### ISSUE THREAD

**Issue by fohvok** [NiLuJe/kfmon#9]
I used the [One-Click Install Package](https://www.mobileread.com/forums/showthread.php?t=314220) and since i [upgraded from 4.x.x to 5.0.178115](https://mytolino.com/software-updates-for-tolino-ereaders/) it does not work anymore when i click on the Koreader Icon (it just opens it as a Picture), and NickelMenu also doesn't appear anymore.

**Comment by fohvok** [comment]
Is there a way to check if it even starts or what the problem is? It looks like in `on-animator.sh` logs get written to `/usr/local/kfmon/kfmon.log`. But i don't know how to access this, since this area is not accessible by USB storage.

**Comment by noiprocks** [comment]
Same

> I used the [One-Click Install Package](https://www.mobileread.com/forums/showthread.php?t=314220) and since i [upgraded from 4.x.x to 5.0.178115](https://mytolino.com/software-updates-for-tolino-ereaders/) it does not work anymore when i click on the Koreader Icon (it just opens it as a Picture), and NickelMenu also doesn't appear anymore.

Same Problem here ... glad to support solving the issue if can contribute anything let me know.

**Comment by NiLuJe** [comment]
(xref to #11)

**Comment by NerdyProjects** [comment]
By the way, I think nickelmenu will be a bit harder to support again. The library import via imageformats doesn't look possible anymore.

**Comment by NiLuJe** [comment]
You mean loading a Qt plugin by passing it as an image format plugin?

**Comment by NerdyProjects** [comment]
Well, nickel does not have the imageformats plugin subfolder anymore.

**Comment by NiLuJe** [comment]
I'll copy @pgaskin & @shermp as I don't remember if we discussed that when we looked at the initial FW images ;).

**Comment by pgaskin** [comment]
> Well, nickel does not have the imageformats plugin subfolder anymore.

Unless something else has changed, we can just create that directory. Regardless, NM will still need work as the toolchain and Qt version has changed.

**Comment by shermp** [comment]
Yeah, and getting a new toolchain up and running probably won't be the most straightforward.

The good news, the new firmware is built using Yocto, rather than the custom thing they were using before. The "bad news", we'll need to try and find suitable layer(s) to be as close as possible to Kobo's.

FWIW, the kernel/u-boot images seem to be based on `(Yocto Project Reference Distro)/4.9/aud8113tp1`

**Comment by beedaddy** [comment]
Perhaps this issue can be closed as completed?

**Comment by NiLuJe** [comment]
Oops, indeed ;).
